site_root/input/resources/profiles/StructureDefinition-kfdrc-condition-no-phi.json

```diff
@@ -0,0 +1,55 @@
+{
+    "resourceType": "StructureDefinition",
+    "id": "kfdrc-condition-no-phi",
+    "url": "http://fhir.kids-first.io/StructureDefinition/kfdrc-condition-no-phi",
+    "version": "0.1.0",
+    "name": "kfdrc-condition-no-phi",
+    "title": "KFDRC Condition Without PHI",
+    "status": "draft",
+    "fhirVersion": "4.0.0",
+    "kind": "resource",
+    "abstract": false,
+    "type": "Condition",
+    "baseDefinition": "http://fhir.kids-first.io/StructureDefinition/kfdrc-condition",
+    "derivation": "constraint",
+    "differential": {
+      "element": [
+        {
+            "id": "Condition.onsetDateTime",
+            "path": "Condition.onsetDateTime",
+            "max": "0"
+        },
+        {
+            "id": "Condition.onsetPeriod",
+            "path": "Condition.onsetPeriod",
+            "max": "0"
+        },
+        {
+            "id": "Condition.onsetString",
+            "path": "Condition.onsetString",
+            "max": "0"
+        },
+        {
+            "id": "Condition.abatementDateTime",
+            "path": "Condition.abatementDateTime",
+            "max": "0"
+        },
+        {
+            "id": "Condition.abatementPeriod",
+            "path": "Condition.abatementPeriod",
+            "max": "0"
+        },
+        {
+            "id": "Condition.abatementString",
```

Comment:
This doesn't seem like a PHI attribute?

Reply:
I wanted to prevent someone putting a date in as string. wasn't sure what the value of string is if age is available.

Reply:
Oh I see.. I think the string is there so that someone could put in a value like Unknown? But then why wouldn't the data type be a CodeableConcept idk 🤷

Reply:
@fiendish the way to prevent a string is by removing that datatype as an allowed type for the abatement[x] field. Also, the cardinality is over the whole abatement[x], not each type. I'll also have to check to see if an *[x] field with cardinality above 1 is allowed to mix the types in the multiple "instances" of that field. I'm assuming it's not allowed. A field has a single type in data and the cardinality and type applies to all of them.

Reply:
i'm curious to see if the StructureDefinition actually validates as is. i'm assuming it doesn't because there really isn't  a "path" of "Condition.abatementPeriod" in the parent profiles and that's usually an error. maybe the *[x] are handled differently